### PR TITLE
Blog app - controllers specs

### PR DESCRIPTION
**Pull request by LokurasRlz**
# In this branch, I have worked on the following things: 

- Added rspec-rails as a gem. ✔️
- Created a Request spec file for Users and Posts controllers.✔️ 

- In each spec, the following was checked:

- If response status was correct.✔️ 
- If a correct template was rendered. ✔️  
- If the response body includes correct placeholder text. ✔️

### CODE REVIEW

File: spec/requests/posts_spec.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      expect(response).to have_http_status(:ok)
+    end
+    it 'should render correct template' do
+      expect(response).to render_template(:index)
+    end
+    it 'body should includes correct placeholder text' do
+      expect(response.body).to include('This is a test')
+    end
+  end
+end
```
Comment (thecodechaser) [inline spec/requests/posts_spec.rb]: - [ ] 6 out of 9 tests are failing, Please make sure all tests are passing successfully.

image

File: spec/requests/posts_spec.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    it 'return correct response' do
+      expect(response).to have_http_status(:ok)
+    end
+    it 'should render correct template' do
+      expect(response).to render_template(:index)
+    end
+    it 'body should includes correct placeholder text' do
+      expect(response.body).to include('This is a test')
+    end
+  end
```
Comment (thecodechaser) [inline spec/requests/posts_spec.rb]: - [ ] The `posts/show` action is missing all tests, Please make sure to add them and all should pass.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @LokurasRlz👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by danmainah — APPROVED** [review]
Hi @LokurasRlz , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**